src/administration/integrations/webhooks.md

```diff
@@ -1,17 +1,97 @@
+
+* `environment.push`: A user has pushed code to a branch, either existing or new.
+* `environment.branch`: A new branch has been created via the UI. (A branch created via a push will show up only as an `environment.push`.)
+* `environment.activate`: A branch has been "activated", and an environment created for it.
+* `environment.initialize`: The master branch of the project has just been initialized with its first commit.
+* `environment.deactivate`: A branch has been "deactivated". The code is still there but the environment was destroyed.
+* `environment.synchronize`: An environment has had its data re-copied from its parent environment.
+* `environment.merge`: A branch was merged through the UI or Platform.sh API. A basic Git merge will not trigger this event.
+* `environment.delete`: A branch was deleted.
+
+* `environment.route.create`: A new route has been created through the UI.  This will not fire for route edits made to the `routes.yaml` file directly.
+* `environment.route.delete`: A route has been deleted through the UI.  This will not fire for route edits made to the `routes.yaml` file directly.
+* `environment.route.update`: A route has been modified through the UI.  This will not fire for route edits made to the `routes.yaml` file directly.
+
+* `environment.variable.create`: A new variable has been created.
+* `environment.variable.delete`: A variable has been deleted.
+* `environment.variable.update`: A variable has been modified.
+
+* `environment.update.http_access`: HTTP access rules for an environment have been modified.
+* `environment.update.smtp`: Sending of emails has been enabled/disabled for an environment.
+* `environment.update.restrict_robots`: The block-all-robots feature has been enabled/disabled.
+* `environment.subscription.update`: The master environment has been resized because the subscription has changed.  There are no content changes.
+
+### `environments`
+
+An array listing the environments that were involved in the activity. This is usually single-value.
+
+### `result`
+
+Whether the activity was completed successfully or not. It should be `success` if all went as planned.
+
+### `created_at`, `started_at`, `completed_at`
+
+These values are all timestamps in UTC.  If you need only a point in time when the action happened, use `completed_at`.  You can also combine it with `started_at` to see how long the activity took.
+
+### `log`
+
+A text description of the action that happened.  This is a human-friendly string that may be displayed to a user but should not be parsed for data as its structure is not guaranteed.
+
+### `payload.environment`
+
+This block contains information about the environment itself, after the action has taken place.  Most notable properties of this key are `name` (the name of the branch) `machine_name` (the name of the environment) and `head_commit` (the Git commit ID that produced the current environment).
```

Comment:
`head_commit` is the head commit SHA of the environment affected by the activity, at the time the activity was created. The way you've described it ("the git commit ID that produced the current environment") makes me interpret it as the commit ID/SHA when the environment was created. What do you think?

Reply:
Updated, and converted it to a list while I was at it.